Source/Core/DolphinWX/NetPlay/NetPlaySetupFrame.cpp

```diff
@@ -19,40 +19,17 @@
 #include "Core/NetPlayServer.h"
 #include "DolphinWX/Frame.h"
 #include "DolphinWX/Main.h"
+#include "DolphinWX/NetPlay/NetPlayLauncher.h"
 #include "DolphinWX/NetPlay/NetPlaySetupFrame.h"
 #include "DolphinWX/NetPlay/NetWindow.h"
 #include "DolphinWX/WxUtils.h"
 
 namespace
```

Comment:
Do we still need this anonymous namespace now that it would only really get rid of one `static`?